app/models/harvest.rb

```diff
@@ -9,7 +9,6 @@ class Harvest < ActiveRecord::Base
   belongs_to :plant_part
   belongs_to :planting
 
-  default_scope { order('created_at DESC') }
```

Comment:
"Harvest ordering lists most recent harvests first" seems upset by this; may need corresponding changes to various controllers, specs, views